ferrischat_webserver/src/entrypoint.rs

```diff
@@ -8,6 +8,7 @@ use crate::not_implemented::not_implemented;
 use crate::users::*;
 use crate::ws::*;
 use actix_web::http::StatusCode;
+use actix_web::middleware::cors::Cors;
```

Comment:
use actix_cors::Cors;

Reply:
That would be a new dependency to add

Reply:
And I think the API is a bit different

Reply:
you can add another dependency

Reply:
> And I think the API is a bit different

wydm?

Reply:
> > And I think the API is a bit different
> 
> wydm?

nvm im dumb

Reply:
yes another dependency `actix-cors = "0.5.4"`